src/signals/incident-management/containers/Dashboard/Dashboard.tsx

```diff
@@ -2,23 +2,21 @@
 // Copyright (C) 2023 Gemeente Amsterdam
 import { useState } from 'react'
 
-import { Row } from '@amsterdam/asc-ui'
-
 import { AreaChart, BarChart, Filter } from './components'
-import { StyledRow } from './styled'
+import { FilterWrapper, StyledRow } from './styled'
 
 const Dashboard = () => {
   const [queryString, setQueryString] = useState<string>('')
 
   return (
     <>
-      <StyledRow data-testid="menu">
+      <FilterWrapper data-testid="menu">
         <Filter callback={setQueryString} />
-      </StyledRow>
-      <Row data-testid="menu">
+      </FilterWrapper>
+      <StyledRow data-testid="menu">
```

Comment:
menu doesnt mean anything here, can be removed?!

Reply:
🥇